Domain/Do-Svyazi.User.Domain/Chats/PersonalChat.cs

```diff
@@ -35,30 +36,47 @@ public class PersonalChat : Chat
         CanDeleteChat = ActionOption.Enabled,
     };
 
-    public PersonalChat(string name, string description)
+    public PersonalChat(MessengerUser user, string name, string description)
         : base(name, description)
     {
+        var admin = new ChatUser
+        {
+            Role = _baseAdminRole,
+            ChatId = Id,
+            User = user,
+        };
+
+        Users.Add(admin);
         BaseAdminRole = _baseAdminRole;
         BaseUserRole = _baseUserRole;
     }
 
     public override void AddUser(ChatUser chatUser)
     {
-        throw new NotImplementedException();
+        if (Users.Count != 1)
+            throw new Do_Svyazi_User_BusinessLogicException($"User {chatUser.User.Name} can't added in chat {Name}");
+
+        Users.Add(chatUser);
     }
 
     public override void RemoveUser(ChatUser chatUser)
     {
-        throw new NotImplementedException();
-    }
+        if (Users.Count != 2)
+            throw new Do_Svyazi_User_BusinessLogicException($"Number users != 2 in chat {Name}");
 
-    public override void AddRole(Role role)
-    {
-        throw new NotImplementedException();
+        if (!Users.Remove(chatUser))
+            throw new Do_Svyazi_User_InnerLogicException($"Error removed user {chatUser.User.Name} in chat {Name}");
```

Comment:
Error removed?)

```suggestion
            throw new Do_Svyazi_User_InnerLogicException($"Chat {Name} does not support adding users");
```

Аналогично остальные